src/main/java/com/redhat/jenkins/plugins/cachet/CachetGatingMetrics.java

```diff
@@ -34,7 +34,7 @@
 
     private String resourceName;
     private Long gateUpdatedTime;
-    private String gatingStatus;
+    private ResourceStatus gatingStatus;
```

Comment:
I am not certain of the top of my head how or why is this serialized exactly, but this is going to break the load of a payload created by an old version. See https://wiki.jenkins.io/display/JENKINS/Hint+on+retaining+backward+compatibility

Reply:
@scoheb